src/test/resources/tdl/runner/ChallengeSession.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Should allow the user to interact with the challenge server
+
+  Background:
+    Given There is a challenge server running on "localhost" port 8222
+    And the challenge server exposes the following endpoints
+      | verb       | endpointMatches        | returnStatus | returnBody                           | acceptHeader  |
+      | GET        | /availableActions/(.*) | 200          | Available actions coming from server | text/coloured |
```

Comment:
I do not think we should have the (.*) here. If I understand correctly, this is a Wiremock thing.

Reply:
(.*) is a regular expression not specific to WireMock, which is why I included it.
If we don't want regular expressions in the cucumber test, I would like it changed from how it was before, from:
`availableActions`
to 
`availableActions/*`
i.e. include an indicator that the journeyId needs to be in the URL.
Whether that is the actual expression or something we substitute out, I don't mind.